docs/source/concepts/LeanElasticPeaksWorkspace.rst

```diff
@@ -48,50 +147,92 @@ LeanElasticPeaksWorkspace Python Interface
 See :class:`IPeaksWorkspace <mantid.api.IPeaksWorkspace>` for
 complete API.
 
+To get peaks from an existing workspace `'name_of_peaks_workspace'`
+
 .. code-block:: python
 
     pws = mtd['name_of_peaks_workspace']
     pws.getNumberPeaks()
     p = pws.getPeak(12)
     pws.removePeak(34)
 
+To create an empty LeanElasticPeaksWorkspace then add peaks to it:
+
+.. testsetup:: createpeaks
+
+   from mantid.kernel import V3D
+
+.. testcode:: createpeaks
+
+   peaks = CreatePeaksWorkspace(NumberOfPeaks=0, OutputType="LeanElasticPeak")
+   p0 = peaks.createPeakQSample([1,1,1])
+   peaks.addPeak(p0)
+   SetUB(peaks, a=2, b=2, c=2)
+   p1 = peaks.createPeakHKL([1,2,3])
+   peaks.addPeak(p1)
+   for n in range(2):
+       print('Peak {} hkl = {h:.0f}{k:.0f}{l:.0f} q_sample = {QSample}'.format(n, **peaks.row(n)))
+
+Output:
+
+.. testoutput:: createpeaks
+
+   Peak 0 hkl = 000 q_sample = [1,1,1]
+   Peak 1 hkl = 123 q_sample = [6.28319,9.42478,3.14159]
+
+
```

Comment:
Method `LeanElasticPeaksWorkspace.getPeak()` is not explained and I think it's relevant to say that the statement `p = peaks.getPeak(n)` will give you a _reference_ to the peak and not a copy of the peak, so that `p.setH(-2)` will change not just `p` but also the cell at column `H` and row `n` of workspace `peaks`.